packages/itwinui-css/src/breadcrumbs/breadcrumbs.scss

```diff
@@ -38,28 +38,27 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   line-height: var(--iui-component-height);
+  border: none;
 
-  &:where(a) {
-    &:focus {
-      outline-offset: -1px;
-    }
+  &,
+  > * {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  &:where(a):focus {
+    outline-offset: -1px;
   }
```

Comment:
To confirm, is this needed? I think when looking at this together, we felt that it is already handled by `iui-field`?

Reply:
I think this whole thing could be simplified?

![](https://github.com/iTwin/iTwinUI/assets/9084735/151ec9af-68eb-4848-9dc7-d12cea16d022)